src/block.js

```diff
@@ -3,7 +3,22 @@
  */
 import { __, sprintf } from '@wordpress/i18n';
 
-export default function Block( { currentTotal, freeShippingFrom } ) {
+export default function Block( { freeShippingFrom, cart } ) {
+	let currentTotal;
+	let currencyPrefix;
+	let currencySuffix;
+
+	if ( undefined === cart ) {
+		currentTotal = 45;
+		currencyPrefix = '$';
+		currencySuffix = '';
+	} else {
+		currentTotal = cart.cartTotals.total_price / 100;
+		currencyPrefix = cart.cartTotals.currency_prefix;
+		currencySuffix = cart.cartTotals.currency_suffix;
+		console.log( cart );
+	}
+
```

Comment:
What's all of this logic?

Reply:
To avoid having this logic, would you suggest not to use `cart.cartTotals.total_price` for now, and add that with the price formatting PR?